asapdiscovery-data/asapdiscovery/data/schema_v2/target.py

```diff
@@ -1,10 +1,18 @@
 from pathlib import Path
 from typing import Any, Dict, Optional, Tuple, Union  # noqa: F401
 
-from asapdiscovery.data.openeye import oechem, oemol_to_pdb_string, pdb_string_to_oemol
+from asapdiscovery.data.openeye import (
+    bytes64_to_oedu,
+    load_openeye_design_unit,
+    oechem,
+    oedu_to_bytes64,
+    oemol_to_pdb_string,
+    pdb_string_to_oemol,
+    save_openeye_design_unit,
+)
 from pydantic import Field, root_validator
 
-from .dynamic_properties import TargetType
+from ..postera.manifold_data_validation import TargetTags
 from .schema_base import (
```

Comment:
is there a good reason to use relative imports here? don't we want to stick to `asapdiscovery.data...` as much as possible?

Reply:
Yep sure, can fix.

Reply:
Done!